frappe_pwa/frappe_pwa/doctype/web_app_manifest/test_web_app_manifest.py

```diff
@@ -6,5 +6,12 @@
 import frappe
 import unittest
 
+from frappe_pwa.frappe_pwa.doctype.web_app_manifest.web_app_manifest import configure_pwa
+
+
 class TestWebAppManifest(unittest.TestCase):
-	pass
+    def test_configure_pwa(self):
+        configure_pwa()
+        ws = frappe.get_doc('Website Settings')
+        ws.head_html.replace('<link href="/assets/frappe_pwa/manifest.json" rel="manifest">', '')
+        ws.save()
```

Comment:
Good thing to add tests, but what is this ??

Also, no asserts means no actual test...